.circleci/config.yml

```diff
@@ -87,6 +87,35 @@ jobs:
             - deps
             - ~/.mix
 
+  excoveralls:
```

Comment:
Can you add Artifacts here so we can view the HTML geneated report? Something like:

```yaml
      - store_artifacts:
          path: ~/repo/coverage.json
          destination: coverage.json
      - store_artifacts:
          path: ~/repo/coverage
          destination: coverage
```